Makefile

```diff
@@ -76,7 +76,7 @@ serve:
 	node server.js
 
 test:
-	./node_modules/.bin/jasmine-node test/katex-spec.js
+	./node_modules/.bin/jasmine-node test/*-spec.js
```

Comment:
I usually do something like `find test/ | grep -- '-spec.js' | xargs ./node_modules/.bin/jasmine-node` for this, so if we ever split things up enough that they need directories then this Just Works. This is fine for now, though!